src/addresses/index.ts

```diff
@@ -0,0 +1,20 @@
+import devAddresses from './addresses.json';
+import testnetAddresses from './addresses.testnet.json';
+import mainnetAddresses from './addresses.mainnet.json';
+
+let addresses: any;
+switch (process.env.INDEXER_ENV) {
+    case "featurenet":
```

Comment:
nit: I guess this is not necessary and indexer could have `devAddresses` always by default.